### PR TITLE
Add Ui to DG

### PR DESCRIPTION
**Pull request by nikkiDEEE**
The Ui sequence diagram is super basic rn, do you think it should include more stuff like an example of how FilterByImportanceCommand object is created by clialgo->parser and how that object calls Ui methods (incl buffer etc)? Lmk, thanks!

### CODE REVIEW

**Comment by nikkiDEEE** [comment]
Improve Ui message output for empty topics when filtering (issue #201)

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -375,6 +387,18 @@ The following sequence diagram shows how the export feature works.
 
 ![](sequence-diagrams/diagrams/Export.png "Export Sequence Diagram")
 
+### Ui
+#### Current implementation
+
+All UI interactions are taken care of by the Ui class. It is responsible for taking in user inputs and giving text-ui
+outputs to provide guidance and a pleasant user experience overall.
+
+The methods in the Ui class use "System.out.println()" from the Java Standard Library to display output to the user.
```
Comment (heejet) [inline docs/DeveloperGuide.md:396]: Also can say it uses the `Scanner` to read in inputs from the user

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+remote_theme: pages-themes/cayman@v0.2.0
+markdown: kramdown
+plugins:
+  - jekyll-remote-theme # add this line to the plugins list if you already have one
```
Comment (heejet) [inline docs/_config.yml:4]: This one we KIV first once we get the content done and when we decide on a theme then we include this

**Review by heejet — CHANGES_REQUESTED** [review]

**Comment by nikkiDEEE** [comment]
updated ui seq diagram and removed config.yml file

**Review by heejet — APPROVED** [review]

**Pull request merged**